packages/visx-text/src/Text.tsx

```diff
@@ -138,7 +138,7 @@ class Text extends React.Component<TextProps, TextState> {
   }
 
   updateWordsWithoutCalculate(props: TextProps) {
-    const words = props.children ? props.children.toString().split(/(?:(?!\u00A0+)\s+)/) : [];
+    const words = (props.children !== null && props.children !== undefined) ? props.children.toString().split(/(?:(?!\u00A0+)\s+)/) : [];
```

Comment:
`!=` I think is still ideal, but there is another lint rule which favors **not** having negated conditions (because they are harder to understand logically). So I think this should fix things:

```suggestion
    const words = props.children == null ? [] : props.children.toString().split(/(?:(?!\u00A0+)\s+)/);
```